build/mvn

```diff
@@ -48,11 +48,11 @@ install_app() {
     # check if we already have the tarball
     # check if we have curl installed
     # download application
-    [ ! -f "${local_tarball}" ] && [ -n "`which curl 2>/dev/null`" ] && \
+    [ ! -f "${local_tarball}" ] && [ -n "`type curl 2>/dev/null`" ] && \
```

Comment:
FWIW, the approach recommended in [this answer](http://stackoverflow.com/a/677212/877069), which I agree with, is to use `command -v`, though honestly one way or the other it doesn't seem like a big deal.